todo/todoapp/views.py

```diff
@@ -16,4 +16,10 @@ class CreateView(generics.ListCreateAPIView):
 
     def perform_create(self, serializer):
         """Save the post data when creating a new Todolist item."""
-        serializer.save()
\ No newline at end of file
+        serializer.save()
+
+class ToDoDetailsView(generics.RetrieveUpdateDestroyAPIView):
+    """GET, PUT and DELETE  existing Todolist"""
```

Comment:
Make the docstrings more descriptives. Like the class handles getting a single product editing the product and deleting the product